autoreduce_frontend/reduction_viewer/views.py

```diff
@@ -250,6 +251,19 @@ def run_summary(request, instrument_name=None, run_number=None, run_version=0):
         if reduction_location and '\\' in reduction_location:
             reduction_location = reduction_location.replace('\\', '/')
 
+        data_location_list = run.data_location.all()
+        data_location = ""
+        if data_location_list:
+            data_location = data_location_list[0].file_path
+            print(data_location)
+
+        format = request.GET.get('format', "")
+        if format != "":
+            if format == "Linux":
+                data_location = windows_to_linux_path(data_location)
+            elif format == "Windows":
+                data_location = linux_to_windows_path(data_location)
```

Comment:
`format` is a built-in function, might not want to shadow it. 

Is `if format != "":` necessary when the `get()` method before it defaults to an empty string anyway? Can just leave that line out.